pnpm-lock.yaml

```diff
@@ -2080,6 +2080,9 @@ packages:
     resolution: {integrity: sha512-R4nPAVTAU0B9D35/Gk3uJf/7XYbQcyohSKdvAxIRSNghFl4e71hVoGnBNQz9cWaXxO2I10KTC+3jMdvvoKw6dQ==}
     engines: {node: '>=6'}
 
+  package-json-from-dist@1.0.0:
```

Comment:
This is introduced by glob@10.4.2

https://github.com/isaacs/node-glob/compare/v10.4.1...v10.4.2